README.md

```diff
@@ -90,6 +101,10 @@ git commit -am "I still do!"            # A combination of the above
 git commit --amend                      # Re-do the commit message of the previous commit (don't do this after pushing!)
                                         #   We _never_ change "public history"
 
+git reset <commit_hash>                 # Undo all commits after <commit_hash> (local changes are preserved)
+git reset --soft <file>                 # Unstage a staged file leaving in working directory without losing any changes. 
```

Comment:
This is not actually an allowed syntax/form of reset. You can use git reset with a file name, but the  `--soft`, `--mixed`, `--hard` and more can only be used without a <pathspec>